templates/yamaha_show_ip_route.textfsm

```diff
@@ -0,0 +1,18 @@
+# You need to set "console character ascci".
+Value NETWORK (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|default)
+Value MASK (\d{1,2})
+Value NEXTHOP_IP (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|-)
+Value NEXTHOP_IF (\S+)
+Value PROTOCOL (\S+)
+Value ADDITIONAL_INFO (\S*)
+
+Start
+  ^Destination\s+Gateway\s+Interface\s+Kind\s+Additional\s+Info -> Routes
+  ^Destination.* -> Routes
```

Comment:
Sorry I was hoping it would delete L11. Do you mind deleting L11 and then we should be good to merge this in.

Reply:
OK. Thank you.